app/controllers/admin/adventure_missions_controller.rb

```diff
@@ -40,7 +40,8 @@ def destroy
     AdventureMissionService.shift_index_on_destroy(@adventure, @adventure_mission.index.to_i)
 
     if @adventure_mission.destroy!
-      redirect_to(admin_adventure_adventure_missions_path(@adventure), notice: alert_destroy(Adventure))
+      redirect_to(admin_adventure_adventure_missions_path(@adventure),
+                  notice: alert_destroy(AdventureMission))
     else
       render :edit, status: 422
     end
```

Comment:
I'm not sure I understand what you mean here. I don't know if we want to disallow a delete just because it is locked since admins might want to delete locked adventure missions

This comment, however, led me to implementing safeguards so one cannot delete an adventure mission if someone has finished it. Or delete an adventure when it has adventure missions. I think this is a reasonable safeguard so nothing bad happens!